src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    public static final String DB_URL = "jdbc:mysql://localhost:3306/taxi_db";
+    public static final String DRIVER = "com.mysql.cj.jdbc.Driver";
+    public static final String USER = "root";
+    public static final String PASSWORD = "vityaak228";
+
+    static {
+        try {
+            Class.forName(DRIVER);
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load JDBC driver for MySQL", e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't load " + DRIVER + " driver", e);
```

thus, if you change the driver over time, you will not need to change the exception text

Reply:
fixed